elixir/apps/domain/test/support/fixtures/gateways.ex

```diff
@@ -110,6 +110,34 @@ defmodule Domain.Fixtures.Gateways do
     %{gateway | online?: false}
   end
 
+  def create_internet_gateway(attrs \\ %{}) do
```

Comment:
The new trigger actually caught tests we had where we were inserting internet resources tied to non-internet sites.

This fixes those.